foundational-cxx-support-matrix.md

```diff
@@ -30,6 +30,17 @@ dimension in question.
 [^rhel-7]: We require the [devtoolset-7] toolchain (compiler, linker, build
 tools, etc.) on RHEL 7 and CentOS 7.
 
+[^cmake]: We support the oldest version of CMake that ships with one of the
+supported distros. Currently that is CMake 3.13 as Debian 10 ships with this
+version.
+
+[^gcc]: Once RHEL 7 reaches EOL we will support the oldest version of GCC
+that ships with one of the supported distros.
+
+[^clang]: We support the oldest version of CLang that ships with one of the
```

Comment:
you say `CLang` here, but in the table it is listed as `Clang`.  Not sure which is correct

Reply:
Thanks.  Fixed, it should be `Clang`.